### ISSUE TITLE
In-place countdown timer instead of static message  Waiting for x seconds, Sleeping for y seconds

### ISSUE THREAD

**Issue by fortunewalla** [rojter-tech/pluradl.py#42]
Present Use: static time for message 'Waiting for 5 seconds' and 'Sleeping for 129 seconds'
Feature request: Only the time part of the message counts down. The message & counter stays on the same line.

Thanks, the tool works perfectly. People need to keep 30min gaps between each course download else website will block you.

**Comment by rojter-tech** [comment]
@fortunewalla Thank you, I save this issue as a reminder. You think 30 min is minimum?

**Comment by fortunewalla** [comment]
> 
> 
> @fortunewalla Thank you, I save this issue as a reminder. You think 30 min is minimum?

Thanks for accepting the request. 

You are doing an amazing work helping people go offline with this knowledge. Very helpful in places with less reliable Internet.

Based on my particular account, the script fails less if I wait for 20-30min. I think they have made their security software more sensitive because of many high activity accounts.

**Comment by Bigemul** [comment]
How many instances do you think we can run in parallel?

**Comment by fortunewalla** [comment]
vary

> 
> 
> How many instances do you think we can run in parallel?

depends on internet connection. 
slow connection max is 2
fast connection: max is 4 (but wait 1 hour after each batch)
fast connection: safe is 2 for continuous learning

Sometimes buffering fails on the website and the video gets stuck. So need to resume again.

**Comment by Bigemul** [comment]
> fast connection: max is 4 (but wait 1 hour after each batch)

1 hour between two courses?

> fast connection: safe is 2 for continuous learning

What do you mean?

**Comment by fortunewalla** [comment]
> 1 hour between two courses?
Yes. I tried four at the same time. Then gave one hour break. Then again four simultaneously. I am guessing PL is monitoring the download throughput. So each break decreases the average value.
> What do you mean?
I found that PL servers fail to stream the video sometimes and it gives 429 error. So for a slow connection try 2 continuous. If fast connection 4 with break.
